src/transaction/TransactionController.ts

```diff
@@ -434,6 +437,7 @@ export class TransactionController extends BaseController<TransactionConfig, Tra
       status: TransactionStatus.unapproved as TransactionStatus.unapproved,
       time: Date.now(),
       transaction,
+      confirmedLocal: confirmedLocalDevice,
```

Comment:
it might be worth having something more expressive here, in case we start tracking more information later. For example, it might be nice to know whether a transaction was confirmed on the extension and transferred here via the extension sync.

Maybe something like `confirmedDevice` would be appropriate here? :thinking: To represent the device it was confirmed on.

Reply:
Good point. I created an enum that gives you three options. Let me know if you think that is too restrictive.

Reply:
Looks good!

Does the name `confirmedLocal` still make sense? As it's now used to represent when transactions were confirmed elsewhere as well.

Reply:
Updated to a bit better descriptor: `deviceConfirmedOn`

Reply:
Thanks! That seems good to me